src/s3/activation.ts

```diff
@@ -4,19 +4,25 @@
  */
 
 import * as vscode from 'vscode'
+import * as telemetry from '../shared/telemetry/telemetry'
+import * as path from 'path'
 import { copyPathCommand } from './commands/copyPath'
 import { createBucketCommand } from './commands/createBucket'
 import { createFolderCommand } from './commands/createFolder'
 import { deleteBucketCommand } from './commands/deleteBucket'
 import { deleteFileCommand } from './commands/deleteFile'
 import { downloadFileAsCommand } from './commands/downloadFileAs'
-import { uploadFileCommand } from './commands/uploadFile'
+import { uploadFileCommand, getFileToUpload, promptUserForBucket } from './commands/uploadFile'
```

Comment:
these imports wouldn't be needed if the logic were moved into `uploadFileCommand`

Reply:
The reason why it was broken down into different methods is to make it easier to control the flow of the 'wizard', also to make testing easier by having to test less things.